src/PyCall.jl

```diff
@@ -708,6 +708,14 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
 end
 
 #########################################################################
+# Expose Python docstrings to Julia doc system
+
+if isdefined(Docs, :getdoc)  # Introduced in Julia .6
+    Docs.getdoc(o::PyObject) = Markdown.parse(o[:__doc__])
```

Comment:
As a result, I think we may have no choice but just to use the plain text string.  I doubt there is any reliable way to convert docstrings to good-looking Markdown.  (The doc2md package seems specialized for docstrings written in a particular way.)